Mathlib/Topology/Constructions.lean

```diff
@@ -491,6 +491,10 @@ theorem continuous_swap : Continuous (Prod.swap : α × β → β × α) :=
   continuous_snd.prod_mk continuous_fst
 #align continuous_swap continuous_swap
 
+lemma isClosedMap_swap : IsClosedMap (Prod.swap : α × β → β × α) := fun s hs ↦ by
+  rw [image_swap_eq_preimage_swap]
+  exact hs.preimage continuous_swap
```

Comment:
I think you can inline a proof using [Homeomorph.prodComm](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Homeomorph.html#Homeomorph.prodComm) with [Homeomorph.isClosedMap](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Homeomorph.html#Homeomorph.isClosedMap) without needing adding this lemma.

Reply:
This file doesn't know about `Homeomorph`s (yes, I think that we should reorder imports but I would prefer to do it later).

Reply:
I agree that we should reorder files, but also that this is fine for now.